.github/workflows/redteam.yml

```
@@ -0,0 +1,65 @@
+          if [ "$HITS" -gt 0 ]; then
+            echo "::error::endpoint keyword leakage: $HITS hits (contracts must ship encrypted per docs/HARDENING.md M2)"
+            SCORE=1
+          fi
+          # 2) no key material / PEM markers
+          if strings /tmp/mediactl-audit | grep -qE "BEGIN (RSA )?PRIVATE KEY|BEGIN OPENSSH PRIVATE KEY"; then
+            echo "::error::key material markers present in artifact"
+            SCORE=1
+          fi
+          # 3) high-entropy region count (source binary should be low-entropy)
+          ENTROPY=$(python3 - "$(stat -c%s /tmp/mediactl-audit)" /tmp/mediactl-audit <<'PYEOF'
+import sys, math, collections
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)